tlm_teamd/values_store.h

```diff
@@ -41,11 +41,13 @@ class ValuesStore
     std::vector<std::string> update_storage(const HashOfRecords & storage);
     void update_db(const HashOfRecords & storage, const std::vector<std::string> & keys_to_refresh);
     void extract_values(const std::string & lag_name, json_t * root, HashOfRecords & storage);
+    void extract_values_static(const std::string & lag_name, json_t * root, HashOfRecords & storage);
 
     HashOfRecords m_storage;  // our main storage
     const swss::DBConnector * m_db;
 
     const std::vector<std::pair<std::string, ValuesStore::json_type>> m_lag_paths = {
+        { "setup.runner_name",           ValuesStore::json_type::string  },
         { "setup.kernel_team_mode_name", ValuesStore::json_type::string  },
```

Comment:
This means that existing configuration (json files) will have to change to accommodate for this new static lag?
Isn't there another way, for example have the LACP to be default, and have no runner name
and only for static lag config we'll have this "Static" runner name